source/adminguide/virtual_machines.rst

```diff
@@ -325,36 +325,22 @@ To access a VM through the CloudStack UI:
-scenarios.
+Every guest VM has an internal name. The host uses the internal name to identify the guest VMs. CloudStack gives you an option to provide a guest VM with a  name. You can set this name as the internal name so that the vCenter can use it to identify the guest VM. A new global parameter, vm.instancename.flag, has now been added to achieve this functionality.
+
+The default format of the internal name is i-<user\_id>-<vm\_id>-<instance.name>, where instance.name is a global parameter. However, If vm.instancename.flag is set to true, and if a name is provided during the creation of a guest VM, the name is appended to the internal name of the guest VM on the host. This makes the internal name format as i-<user\_id>-<vm\_id>-<name>. The default value of vm.instancename.flag is set to false. This feature is intended to make the correlation between instance names and internal names easier in large data center deployments.
+
+The following table explains how a VM name is displayed in different scenarios.
 
 .. cssclass:: table-striped table-bordered table-hover
 
-============================= ======================= ==================== ===================================== ==========================
-User-Provided Display Name    vm.instancename.flag    Hostname on the VM   Name on vCenter                       Internal Name
-============================= ======================= ==================== ===================================== ==========================
-Yes                           True                    Display name         i-<user\_id>-<vm\_id>-displayName     i-<user\_id>-<vm\_id>-displayName
-No                            True                    UUID                 i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-Yes                           False                   Display name         i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-No                            False                   UUID                 i-<user\_id>-<vm\_id>-<instance.name> i-<user\_id>-<vm\_id>-<instance.name>
-============================= ======================= ==================== ===================================== ==========================
+============================= ======================= ====================== ====================== ====================== ===================================== ==========================
+User-Provided Display Name    vm.instancename.flag    User                   Display Name           Hostname on the VM     Name on vCenter                       Internal Name
+============================= ======================= ====================== ====================== ====================== ===================================== ==========================
+Yes                           True                    Name                   Display name           Name                   i-<user\_id>-<vm\_id>-name            i-<user\_id>-<vm\_id>-name
```

Comment:
can you please change the Name and display name to <Name> and <Display name> because those are user defined